contracts/src/TeleporterTokenSource.sol

```diff
@@ -275,6 +276,7 @@ abstract contract TeleporterTokenSource is
 
         uint256 adjustedAmount;
         uint256 feeAmount = input.primaryFee;
+        address originBridgeAddress_ = address(this);
```

Comment:
Is this extra variable/assignment needed? I think the `originBridgeAddress` passed in to the internal `_sendAndCall` will already have the proper value in all cases.

Reply:
Didn't check the values passed by the callers closely enough. Removed as suggested.